docs/audio.rst

```diff
@@ -14,7 +17,7 @@ which means that it can reproduce frequencies up to 3.9kHz.
 Functions
 =========
 
-.. py:function:: play(source, wait=True, pin=pin0, return_pin=None)
+.. py:function:: play(source, wait=True, pin=pin_speaker, return_pin=None)
```

Comment:
Des this work? https://microbit-micropython--692.org.readthedocs.build/en/692/audio.html#audio.play

 ![image](https://user-images.githubusercontent.com/31242877/100358894-b88e7d80-2fee-11eb-8814-71dc88b94298.png)

Reply:
Do you know if there is a way to add a note? Maybe like a `(V1)` or `(V2)` at the end?

I know exactly this might not be difficult (I just edited the HTML in the dev tools), but it's just to illustrate what I meant: 
![image](https://user-images.githubusercontent.com/29712657/100360467-e379d100-2ff0-11eb-8520-b22de085f8be.png)

There might also be other ways to identify the different signatures? I'm very open for suggestions.

Reply:
I haven't found a way to do this inline. Will keep looking